### ISSUE TITLE
Changes IntWisAs

### ISSUE THREAD

**Issue by JWocky** [FGMEMBERS/Jumbolino#4]
1.) Engine sound now before engines are started, but parts of it (wave) not anymore after they are running

2.) About a dozen object not found for animation connected to new instruments

3.) panel expansion and angle of the new instruments are inconsistent.

Haven't even looked yet at your "AP bug fixes" yet, but if that are that way, I am scared you are breaking a plane that flew now fine for years ande would ask you politeily to make your own fork. Thank you!

**Comment by ghost** [comment]
Sorry for the late reply, I just saw this now. I hope I didn't break anything and my apologies if I did.

**Comment by JWocky** [comment]
No irreparable damage done yet, at least as far as I know. 

1.) The changed sound, you need to check the sounds-xml, I guess, you have somewhere a condition not included or forgot to exchange the filename somewhere.

2.) There is something with tour instrument models, the animations say, they don#t find them.

3.) I didn't find time to look for the AP bug fixes ... but since the AP is linked in from JPack, I am more than nervous if someone says, he fixed bugs in the AP without touching JPack ... I mean, whatever you fixed, it can't be the AP, right?

**Comment by ghost** [comment]
I don't recall fixing or adding anything to the AP (and I'm assuming AP refers to the autopilot).

**Comment by JWocky** [comment]
No, I wrote that wrong, sorry, my bad, my apologies (someone played at the same time with an AP on another plane...)

So, the other problems are minor. You know, how to get that under control? I am not much of a cockpit builder myself.

**Comment by ghost** [comment]
I'm sure I can fix it, just not at this moment. (Again, sorry if I messed up anything important)

**Comment by JWocky** [comment]
Since the problems are not inflicting the flyability, I suggest, we leave the changes in and care for the little glitches as we find time (I have no time atm either). I mean, by all means, having more of a cockpit would be nice with that plane, she is one of my go to planes (any chances to get a flap and gear indicator there)?
